commands/serverless_util.go

```diff
@@ -112,11 +110,7 @@ func (c *CmdConfig) PrintServerlessTextOutput(output do.ServerlessOutput) error
 }
 
 func hashAccessToken(c *CmdConfig) string {
-	token := c.getContextAccessToken()
-	hasher := sha1.New()
-	hasher.Write([]byte(token))
-	sha := hasher.Sum(nil)
-	return hex.EncodeToString(sha[:4])
+	return do.HashAccessToken(c.getContextAccessToken())
```

Comment:
Most of the logic is relocated to the do package but call sites remain in the commands package, hence the original function is retained.